header.php

```diff
@@ -9,8 +9,17 @@
  * @package Checathlon
  */
 
+$body_classes = get_body_class();
+
+$html_class = ' class="no-js"';
+$body_class = ' class="' . implode( ' ', $body_classes ) . '"';
+if ( checathlon_is_amp() ) {
+	$html_class .= ' [class]="\'no-js\' + ( navMenuToggledOn ? \' disable-scroll\' : \'\' )"';
```

Comment:
This could be made a bit cleaner by doing:

```suggestion
	$html_class .= sprintf( ' [class]="%s"', esc_attr( "'no-js' + ( navMenuToggledOn ? ' disable-scroll' : ''" ) ) );
```